config/config.exs

```diff
@@ -2,4 +2,5 @@ use Mix.Config
 
 if Mix.env == :test do
   config :ex_unit, :assert_receive_timeout, 1000
+  config :postgrex, :pgpass, "test/support/pgpass"
```

Comment:
This line is not for selecting the pgpass file at runtime. It's purely for testing purposes based on an article José wrote about testing at boundaries. http://blog.plataformatec.com.br/2015/10/mocks-and-explicit-contracts/

I agree with your recommendation on using PGPASSFILE from the env. Would this require a change to the Travis setup to make the tests pass, or do I just System.put_env on test runs?